source/ocsp-support/ocsp-support.rst

```diff
@@ -100,9 +100,10 @@ invalid, the driver SHOULD end the connection.
     attempt to validate the status of the unvalidated certificates
     using the cached CRLs.
 
-7.  If the server’s certificate remains unvalidated and that certificate
-    has an OCSP endpoint, the driver SHOULD reach out to the OCSP
-    endpoint specified and attempt to validate that certificate.
+7.  If the server's certificate remains unvalidated and that certificate
+    has a list of OCSP responder endpoints, the driver SHOULD reach out to the
+    responders in a loop until a responder returns a response that concretely
+    marks the certificate status as good or revoked.
```

Comment:
Hm, should we also clarify what happens when the OCSP responder is unavailable? Soft-fail and connect after a specified timeout period seems reasonable: what does Go/Python currently do in this particular case?

Reply:
Right now I've got a 5 second timeout on the request to the OCSP URI. If the timeout expires I soft fail.

Reply:
For Go, I'm using the user-supplied context, but that need not have a timeout associated with it (the user could pass in something like `context.TODO()` which never expires or gets cancelled). I think it's a reasonable idea for drivers to enforce a default timeout. The Go code would be similar to our server selection code, as we would check both the user-supplied context and our own timeout for expiration/cancellation.

Reply:
Do we agree that 5 seconds is reasonable?

Reply:
5s sounds fine to me! I've also pinged Shreyas and if the shell is doing something drastically different we can circle back to this later.

Reply:
Added a requirement to set a 5s timeout on OCSP requests in my latest commit.